ui/src/app/components/dataflow-form/dataflow-form.component.ts

```diff
@@ -11,6 +11,8 @@ import ISpannerConfig from 'src/app/model/spanner-config';
 })
 export class DataflowFormComponent implements OnInit {
   dataflowForm: FormGroup
```

Comment:
Let's rename this form to something more meaningful now that we have two other forms all of which contribute to dataflow configuration.